src/main/java/com/razorpay/ApiUtils.java

```diff
@@ -159,4 +165,15 @@ private static Response processRequest(Request request) throws RazorpayException
   static void addHeaders(Map<String, String> header) {
     headers.putAll(header);
   }
+  
+  private static X509TrustManager createDefaultTrustManager() throws NoSuchAlgorithmException, KeyStoreException {
+    TrustManagerFactory trustManagerFactory = TrustManagerFactory.getInstance(TrustManagerFactory.getDefaultAlgorithm());
+    trustManagerFactory.init((KeyStore) null);
+    TrustManager[] trustManagers = trustManagerFactory.getTrustManagers();
+    if (trustManagers.length != 1 || !(trustManagers[0] instanceof X509TrustManager)) {
```

Comment:
Why is the length check required?

Reply:
copied the code from : https://square.github.io/okhttp/3.x/okhttp/okhttp3/OkHttpClient.Builder.html#sslSocketFactory-javax.net.ssl.SSLSocketFactory-javax.net.ssl.X509TrustManager-